src/utils/HypervBoxUtil.js

```diff
@@ -31,19 +28,19 @@ var HypervBox = {
   */
 
   hasAdminRights: function() {
-    return util.execFile([this.command(), 'Get-VMHostSupportedVersion']).then(stdout => {
```

Comment:
That function is not available for all installations. It is present on 50% of the machines I've tested.